server/lorax_server/models/custom_modeling/flash_mistral_modeling.py

```diff
@@ -260,6 +260,14 @@ def __init__(
             )
         self.num_heads = self.num_heads // weights.process_group.size()
         self.num_key_value_heads = config.num_key_value_heads // weights.process_group.size()
+        if paged_attention.is_fp8_supported() and config.quantize and config.quantize.endswith('_kv'):
```

Comment:
This is a little hacky. Can we have a function like `is_fp8_kv` or something instead of checking for a suffix?

Reply:
Done.